src/updatechecker.h

```diff
@@ -70,7 +70,8 @@ class UpdateChecker : public Thread
     virtual bool ShouldAutomaticallyInstall() const { return false; }
 
 protected:
-    virtual void Run();
```

Comment:
The use of `virtual` here is intentional, to clearly mark it as overriden (modern C++ would use `override`, I don’t use it for compatibility yet).